### PR TITLE
Update 10/11

### PR DESCRIPTION
**Pull request by AlEsmail**
Observed 3 developers use the site. I've made some text edits based on their behavior, enthusiasm (or lack there of) and feedback.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,22 +45,22 @@
...
 </div></div>
 <div class="blocks" ><div class="image" style="background-image:url({{ site.url }}{{ site.baseurl }}assets/style/img/siafi.svg);"></div></div>
 </div>
 <div class="inner_info"  style="background-image:url({{ site.url }}{{ site.baseurl }}assets/style/img/eig.svg);">
 <div class="blocks"><div class="image" style="background-image:url({{ site.url }}{{ site.baseurl }}assets/style/img/eig.svg);"></div></div>
 <div class="blocks textblocks"><div class="text">
-<span class="t_i1">Slack and Github Integration</span>
-<span class="t_i2">Notify the entire developer team by Slack so that action can be taken immediately. Set WoTT to open Github issues and seamlessly incorporate security upgrades your CI/CD pipeline.</span>
+<span class="t_i1">Github Integration</span>
+<span class="t_i2"> Set WoTT to open Github issues and seamlessly incorporate security upgrades into your CI/CD pipeline.</span>
```
Comment (vpetersson) [inline index.html]: Remove leading space.

File: index.html [hunk 2]
```diff
@@ -18,19 +18,19 @@
 </div>
 <div class="wff_90">
 <span class="t_hm">our products</span>
-<span class="t_h1">Fix server security with step-by-step instructions</span>
-<span class="t_h2">Get security alerts delivered to your Inbox or Slack as they are discovered. Turn your DevOps team into a DevSecOps team.</span>
+<span class="t_h1">A lightweight agent installed on your server protects your data and workloads</span>
```
Comment (vpetersson) [inline index.html]: I think this needs to be shrunk a bit.

Comment (AlEsmail) [reply index.html]: Updated the branch. I've shortened it from the first proposed change but it's still a bit longer than the previous line. I think it'll be ok looking at the current site layout.

File: index.html (hunk 1 above)
Comment (AlEsmail) [reply index.html]: Done

**Pull request merged**